Assets/_Project/Code/Scripts/Lobby/LobbyController.cs

```diff
@@ -24,6 +97,40 @@ public class LobbyController : NetworkBehaviour
 
     void Start()
     {
+        //to delete after testing start
+
+
+
+
+
+
+         
+            NetMan = this.NetworkManager;
+        Debug.Log(NetMan);
+            General_Game_Data.NetMan = NetMan; 
+             BiddingWarButton.onClick.AddListener(Load_BW_Host);
+
+            
+
+            if (!IsHost)
+            {
+                Add_TeamRpc(NetMan.LocalClientId);
+
+            }
+            else
+            {
+                Add_TeamRpc(NetMan.LocalClientId);
+                /*
+                Teams.Add(Remaining_Teams[0]);
+                Remaining_Teams.RemoveAt(0);
+                Teams[0].ID = NetMan.LocalClientId;*/
```

Comment:
Nie wiem czy jest to niedokończone czy nie będzie już potrzebne. Jeżeli skończyłeś testowanie to wydaje mi się że można takie rzeczy usunąć